datamonster_api/lib/datamonster.py

```diff
@@ -160,6 +163,40 @@ def get_datasource_by_id(self, datasource_id):
         datasource["uri"] = self._get_datasource_path(datasource_id)
         return self._datasource_result_to_object(datasource, has_details=True)
 
+    def get_datasource_details(self, datasource_id):
+        """Get details (metadata) for the given datasource
+
+        :param datasource_id: The ID of the datasource for which we get the details
+        :return: dictionary object with the datasource details
+        """
+        return self.client.get(self._get_datasource_path(datasource_id))
```

Comment:
I don't really care, but personally I think it's easier to read if this is broken into 2 lines as it was originally.